src/utilities/CMakeLists.txt

```diff
@@ -343,6 +345,13 @@ set(${target_name}_test_src
   bcl/test/BCLMeasure_GTest.cpp
 )
 
+## # Example: Selectively add test files
```

Comment:
@ggartside this was an example if you needed it, but if you want I think it can be removed now.

Reply:
removed in bff04eb459d0bd3c3e4f2588239ee3f348159cec